blog/static/.htaccess

```diff
@@ -0,0 +1 @@
+ErrorDocument 404 https://blog.3mdeb.com/404
```

Comment:
@artur-rs according to this topic: https://github.com/gohugoio/hugo/issues/1249 I created extra htaccess file. Can't test this out on local.

Reply:
we should test it on `dev`, so I commited `https://beta.blog.3mdeb.com/404`